pkg/questionModel/confirmation.go

```diff
@@ -0,0 +1,153 @@
+is comprised of two SingleSelectLists: The first is the list of configurations and the second
+is the yes/no option to confirm the launch of the instance.
+*/
+type Confirmation struct {
+	lists      []SingleSelectList
+	choice     string // The chosen option
+	focusIndex int    // The index of the cursor
+	allowEdit  bool   // Whether the configurations list is selectable
+	errorMsg   string // An error message to be presented if a config is selected which cant be reconfigured
+	err        error  // An error caught during the question
+}
+
+// InitializeModel initializes the model based on the passed in question input
+func (c *Confirmation) InitializeModel(input *QuestionInput) {
+	configList := SingleSelectList{}
+	configList.InitializeModel(&QuestionInput{
+		HeaderStrings:  []string{"Configurations", "Values"},
+		QuestionString: "Please confirm if you would like to launch instance with following options:",
+		Rows:           input.Rows,
+		IndexedOptions: input.IndexedOptions,
+	})
+	configList.list.Select(-1)
+
+	yesNoList := SingleSelectList{}
+	yesNoList.InitializeModel(&QuestionInput{
+		IndexedOptions: yesNoOptions,
+		DefaultOption:  cli.ResponseNo,
+		Rows:           CreateSingleLineRows(yesNoData),
+	})
+	c.lists = append(c.lists, configList, yesNoList)
+	c.focusIndex = 1
+}
+
+// Init defines an optional command that can be run when the question is asked.
+func (c *Confirmation) Init() tea.Cmd {
+	return nil
+}
+
+/*
+Update is called when a message is received. Handles user input to traverse through lists and
+select an answer.
+*/
+func (c *Confirmation) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	switch msg := msg.(type) {
+
+	case tea.KeyMsg:
+		switch msg.Type {
+		case tea.KeyCtrlC:
+			c.err = exitError
+			return c, tea.Quit
+
+		case tea.KeyUp:
+			// Decrease the cursor index if there are more elements to move up to, and if they're allowed
+			// to be focused on.
+			if c.focusIndex > -len(c.lists[0].list.Items()) && (c.allowEdit || c.focusIndex > 0) {
+				c.focusIndex--
+			}
+			// Focus the config list and unfocus the yes/no list
+			if c.focusIndex == -1 {
+				c.lists[0].list.Select(len(c.lists[0].list.Items()))
+				c.lists[1].list.Select(c.focusIndex)
+			}
+
+		case tea.KeyDown:
+			// Increase the cursor index if there are more elements to move down to
+			if c.focusIndex < len(c.lists[1].list.Items())-1 {
+				c.focusIndex++
+			}
+			// Focus the yes/no list and unfocus the config list
+			if c.focusIndex == 0 {
+				c.lists[0].list.Select(-1)
+				c.lists[1].list.Select(c.focusIndex)
+				return c, nil
+			}
+
+		case tea.KeyEnter:
+			c.errorMsg = ""
+			// Select an item from the appropriate list
+			if c.focusIndex < 0 {
+				c.lists[0].selectItem()
+				c.choice = c.lists[0].GetChoice()
+				// Set an error message if there is no choice associated with the option
+				if c.choice == "" {
+					c.errorMsg = "This configuration is a question that can be repeated!"
```

Comment:
Feel free to disagree, but I think this is clearer:

```suggestion
					c.errorMsg = "This configuration can't be modified"
```